src/main/scala/com/raphtory/core/actors/AnalysisManager/Tasks/AnalysisTask.scala

```diff
@@ -86,7 +86,7 @@ abstract class AnalysisTask(
     case ClassMissing => log.error("unexpected missing")
     case ClassExists =>
       if (readyCount + 1 == managerCount) {
-        messageToAllReaderWorkers(LoadPredefinedAnalyser(jobId, analyser))
+        messageToAllReaderWorkers(LoadPredefinedAnalyser(jobId, analyser, args))
```

Comment:
why you need this? you just pass the the build analyser instance directly to the target. You do not need to build another one I think